pom.xml

```diff
@@ -141,8 +141,6 @@
     <profile>
       <id>stdbuild</id>
       <activation>
-        <!-- this selects any Java7 JDK or higher, split was needed as
-             checkstyle is incompatible with JDK6 -->
         <jdk>[1.7,)</jdk>
```

Comment:
Should this be changed from 1.6 upwards?

Reply:
If I understand correctly, previously we had this check so this part of the pom file was only active on the non-Java6 builds, and the other part was active on the Java6 build.
Since we don't actually use Java6 anywhere on Travis any more with this PR, wouldn't it be misleading to change this to 1.6? Alternatively I could just set it to
```        <activeByDefault>true</activeByDefault>```
since it is the only profile used on Travis runs now.
What do you think?

Reply:
Yes setting to active by default seems the clearest thing to do here.

Reply:
Done